### PR TITLE
Split jesexercises into more specific and targetted test cases. Closes #951

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: src/main/scala/centaur/test/formulas/TestFormulas.scala [hunk 1]
```diff
@@ -16,6 +16,8 @@ object TestFormulas {
   def runWorkflowUntilTerminalStatus(workflow: Workflow, status: TerminalStatus): Test[SubmittedWorkflow] = {
     for {
       s <- submitWorkflow(workflow)
+      // TODO: To temporarily resolve "eventual consistency" - Cromwell is being changed to help this so we can remove this soon.
+      _ = Thread.sleep(5000)
```
Comment (mcovarr) [inline src/main/scala/centaur/test/formulas/TestFormulas.scala]: you should be able to get rid of this now

**Comment by mcovarr** [comment]
:+1: it looks like there's some new stuff here too?  

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/55/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by cjllanwarne** [comment]
Could be some extras too. I was trying to cover any possible permutation in the intention of the original tests.

File: src/main/resources/standardTestCases/curl.test [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: curl
```
Comment (geoffjentry) [inline src/main/resources/standardTestCases/curl.test:1]: Since this test isn't looking at output from the page, would you agree that #21 could be closed?

**Comment by geoffjentry** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/55/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**